src/lib/TranscriptEditor/TimedTextEditor/WrapperBlock.module.css

```diff
@@ -1,44 +1,96 @@
 @value color-labs-red, color-light-grey, color-mid-grey, color-dark-grey from '../colours.module.css';
 
-.WrapperBlock {
-  height: 100%;
-  width: 100%;
-  margin-bottom: 2em;
-  line-height: 1.2em;
-  overflow: hidden;
-}
+/* Desktop size */
+@media (min-width: 768px) {
 
-.markers {
-  width: 25%;
-  font-size: 0.8em;
-  line-height: 1.2em;
-  float: left;
-}
+  .markers {
+    width: 25%;
+    font-size: 0.8em;
+    line-height: 1.2em;
+    float: left;
+  }
+
+  .WrapperBlock {
+    height: 100%;
+    width: 100%;
+    margin-bottom: 2em;
+    line-height: 1.2em;
+    overflow: hidden;
+  }
+
+  .speaker {
+    display: inline-block;
+    width: 50%;
+    padding-right: 2em;
+    vertical-align: middle;
+    color: color-mid-grey;
+    font-weight: bold;
+    text-transform: uppercase;
+    text-align: right;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .text {
+    width: 75%;
+    font-size: 1em;
+    float: right;
+    clear: right;
+  }
 
-.text {
-  width: 75%;
-  font-size: 1em;
-  float: right;
-  clear: right;
 }
 
-.speaker {
-  display: inline-block;
-  width: 50%;
-  padding-right: 2em;
-  vertical-align: middle;
-  color: color-mid-grey;
-  font-weight: bold;
-  text-transform: uppercase;
-  text-align: right;
-  text-overflow: ellipsis;
-  overflow: hidden;
-  white-space: nowrap;
+/* Mobile devices */
```

Comment:
The WrapperBlock part looks good for the mobile widths 👌 
I would maybe give the html/body a min-width of 320px so it doesn't squash down anymore.

Reply:
not sure I follow?